src/puppetlabs/puppetdb/admin.clj

```diff
@@ -0,0 +1,53 @@
+(ns puppetlabs.puppetdb.admin
+  (:require [clojure.java.io :as io]
+            [clojure.tools.logging :as log]
+            [compojure.core :as compojure]
+            [puppetlabs.kitchensink.core :as kitchensink]
+            [puppetlabs.puppetdb.archive :as archive]
+            [puppetlabs.puppetdb.cheshire :as json]
+            [puppetlabs.puppetdb.cli.export :as export]
+            [puppetlabs.puppetdb.cli.import :as import]
+            [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.middleware :as mid]
+            [puppetlabs.trapperkeeper.core :as trapperkeeper]
+            [ring.middleware.multipart-params :as mp]
+            [clj-time.core :refer [now]]
+            [ring.util.io :as rio]))
+
+(def query-api-version :v4)
+
+(def munge-nodes-data
+  (comp (partial kitchensink/mapvals str)
```

Comment:
It's clearer, faster, and 2 characters shorter to write

(defn munge-nodes-data [nodes]
    (kitchensink/mapvals str
       (select-keys nodes ...)))

Point-free style is great in some places, but I don't think this is it.